### PR TITLE
Wallpaper customization

### PR DESCRIPTION
**Pull request by ader1990**

### CODE REVIEW

File: WinImageBuilder.psm1 [hunk 1]
```diff
@@ -1086,6 +1086,11 @@ function New-WindowsCloudImage {
         Generate-UnattendXml @xmlParams
         Copy-UnattendResources $resourcesDir $image.ImageInstallationType $windowsImageConfig.install_maas_hooks
         Copy-Item $ConfigFilePath "$resourcesDir\config.ini"
+        if ($windowsImageConfig.wallpaper_path -and `
+                (@('.jpg', '.jpeg') -contains (Get-Item $windowsImageConfig.wallpaper_path -ErrorAction SilentlyContinue).Extension)) {
+            Write-Host "Copying custom wallpaper to the image."
```
Comment (costingalan) [inline WinImageBuilder.psm1]: Cosmetic: leave a set of 3 points at the end of the line.

File: WinImageBuilder.psm1 [hunk 2]
```diff
@@ -1086,6 +1086,11 @@ function New-WindowsCloudImage {
         Generate-UnattendXml @xmlParams
         Copy-UnattendResources $resourcesDir $image.ImageInstallationType $windowsImageConfig.install_maas_hooks
         Copy-Item $ConfigFilePath "$resourcesDir\config.ini"
+        if ($windowsImageConfig.wallpaper_path -and `
+                (@('.jpg', '.jpeg') -contains (Get-Item $windowsImageConfig.wallpaper_path -ErrorAction SilentlyContinue).Extension)) {
+            Write-Host "Copying custom wallpaper to the image."
+            Copy-Item $windowsImageConfig.wallpaper_path "$resourcesDir\Wallpaper.jpg" -Force
```
Comment (costingalan) [inline WinImageBuilder.psm1]: I'd put quotes on the wallpaper path knowing that it's user input.

File: WinImageBuilder.psm1 [hunk 3]
```diff
@@ -1086,6 +1086,11 @@ function New-WindowsCloudImage {
         Generate-UnattendXml @xmlParams
         Copy-UnattendResources $resourcesDir $image.ImageInstallationType $windowsImageConfig.install_maas_hooks
         Copy-Item $ConfigFilePath "$resourcesDir\config.ini"
+        if ($windowsImageConfig.wallpaper_path -and `
```
Comment (costingalan) [inline WinImageBuilder.psm1]: Don't we need a Test-Path to check if the file exists?

Comment (ader1990) [inline WinImageBuilder.psm1]: get-item returns $null if the path does not exist

File: WinImageBuilder.psm1 (hunk 2 above)
Comment (ader1990) [inline WinImageBuilder.psm1]: e variablia de PS, nu este bash scripting

**Comment by ader1990** [comment]
golden image still needs to be implemented

**Pull request merged**